### PR TITLE
Fix offset logic for pagination

### PR DESCRIPTION
**Pull request by erdemgezer**

### CODE REVIEW

File: lib/pragmatic_serializer/pagination_json.rb [hunk 1]
```diff
@@ -26,11 +24,12 @@ def first
     end
 
     def next
-      pagination_evaluator.call(limit: limit, offset: offset+1) unless offset >= maximum_offset
+      pagination_evaluator.call(limit: limit, offset: offset + limit)
```
Comment (equivalent) [inline lib/pragmatic_serializer/pagination_json.rb]: why did you remove the maximum offset ?

File: lib/pragmatic_serializer/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module PragmaticSerializer
-  VERSION = "0.2.2"
+  VERSION = "0.2.3"
```
Comment (equivalent) [inline lib/pragmatic_serializer/version.rb]: this should be "0.3" as this is big change

File: lib/pragmatic_serializer/pagination_json.rb (hunk 1 above)
Comment (equivalent) [reply lib/pragmatic_serializer/pagination_json.rb]: i briefly remember I've implemented the Maximum in order to prevent large limits, :thinking:  now that I'm thinking about it it may be a `maximum_limit` not sure what do you think ?

**Comment by equivalent** [comment]
also can you pls pull latest master to your branch ? ...I've fixed the travis CI

File: lib/pragmatic_serializer/version.rb (hunk 2 above)
Comment (erdemgezer) [reply lib/pragmatic_serializer/version.rb]: ok changing to 0.3

File: lib/pragmatic_serializer/pagination_json.rb (hunk 1 above)
Comment (erdemgezer) [reply lib/pragmatic_serializer/pagination_json.rb]: I think it's the user's responsibility to set the maximum offset. We shouldn't be dealing with that in the gem

Comment (equivalent) [reply lib/pragmatic_serializer/pagination_json.rb]: I see, ok I agree  :+1:
